### DISCUSSION TITLE
Sonoff ZigBee Bridge and Home Assistant

### DISCUSSION THREAD

**Discussion by izumin007** [arendst/Tasmota#12610] [Q&A] [answered]
Hi, i'm newbie start HA. i'm stuck here can't move to next step to add Sonoff Zigbee Brigde to HA. 

<img width="273" alt="upload github" src="https://user-images.githubusercontent.com/39380474/125118381-d3df7080-e119-11eb-99eb-3fba51ac0b35.PNG">

My console: 
`backlog rule1 on system#boot do TCPStart 8888 endon ; rule1 1 ; template {"NAME":"Sonoff ZHABridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18} ; module 0`

My template:
{"NAME":"Sonoff ZHABridge","GPIO":[320,5088,0,5120,323,322,0,0,0,0,0,0,32,0],"FLAG":0,"BASE":18}
Tasmota 9.5.0
upgraded last ota tasmota.bin
Any help thank you.

**Comment by ascillato** [comment, marked as answer]
Hi,

In the Tasmota's documentation, the procedure to set up a ZigBee Bridge don't use what you are trying to do.

Have you tried https://tasmota.github.io/docs/Home-Assistant/#zigbee-devices ?
